Geometry/data/gem21.xml

```diff
@@ -427,10 +427,10 @@
     <rSolid name="GEMBox21L"/>
     <rMaterial name="materials:Air"/>
   </LogicalPart>
-  <!-- <LogicalPart name="GEMDisk21LN" category="unspecified">
+  <LogicalPart name="GEMDisk21LN" category="unspecified">
```

Comment:
why is this changed?

Reply:
In the same process of checking what the differences were, I found that it was the only comment that was in the logic part and I removed it to see what happened and I forgot to return the comment, if you want I can remove that part of the code and make the pull request of new

Reply:
this was commented out for a reason